lighthouse-core/test/audits/dobetterweb/has-inspector-issues-test.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const InspectorIssuesAudit =
+  require('../../../audits/dobetterweb/has-inspector-issues.js');
+
+/* eslint-env jest */
+
+describe('Has inspector issues audit', () => {
+  it('passes when no issues are found', () => {
+    const issues = {
```

Comment:
you should just be able to add a `/** @type {LH.Artifacts.InspectorIssues} */` to the top of this rather than having to annotate the individual properties